Adafruit_Thermal.py

```diff
@@ -188,11 +193,11 @@ def __init__(self, bus=1, baudrate=9600, pins=('P1', 'P0'), **kwargs):
 
 	# Sets estimated completion time for a just-issued task.
 	def timeoutSet(self, x):
-		self.resumeTime = utime.ticks_ms() + x*1000
+		self.resumeTime = utime.ticks_add(utime.ticks_ms(), x)
```

Comment:
`ticks_add()` is not implemented in Pycom's micropython. As discussed, we should probably make a hardware-dependent wrapper for it.